### ISSUE TITLE
xml2json library is no longer maintained. use xml2js

### ISSUE THREAD

**Issue by thiccaxe** [StudentVue-Community/StudentVue.js#15]
xml2js also doesn't depend on libexpat, which seems need software installed on windows.

Investigating if this will change the output structure.

**Comment by thiccaxe** [comment]
output structure changes.

even better plan is probably to drop the dependency all together and make dependents roll their own library. The purpose of this library should be _primarily_ dealing with the SOAP nonsense, not xml to json conversion, which is opinionated.

**Comment by thiccaxe** [comment]
see https://github.com/StudentVue/StudentVue.js/tree/remove-xml-parse

**Comment by LoganMD** [comment]
open a pr and ill gladly merge it

**Comment by kaijchang** [comment]
> output structure changes.
> 
> even better plan is probably to drop the dependency all together and make dependents roll their own library. The purpose of this library should be _primarily_ dealing with the SOAP nonsense, not xml to json conversion, which is opinionated.

somewhat fair, but XML is not a format usable in Javascript, JSON is literally Javascript Object Notation. best solution might to be allow user to configure the XML -> JSON converter through options on `StudentVueClient` like in the Python version. don't think it really makes sense to make each user do their own conversion.

**Comment by thiccaxe** [comment]
yep that was an idea I had. the default would be to just return the raw xml.
